src/main.rs

```diff
@@ -693,12 +693,15 @@ fn main() {
 
           dirty = true;
 
+          let ahead_behind = format!("[ahead {}, behind {}]",
+            console::style(project_status.ahead.to_string()).green(),
+            console::style(project_status.behind.to_string()).red());
           let project_line = PROJECT_STYLE.apply_to(format!("project {:64}", project_name));
           let branch = match &project_status.branch {
-            Some(branch) => BRANCH_STYLE.apply_to(format!("branch {}", branch)),
+            Some(branch) => BRANCH_STYLE.apply_to(format!("branch {:32}", branch)),
```

Comment:
I'm not sure there's much point in padding the branch name out: the colors from ahead/behind should be good enough to make it visually clear?

(More importantly, it's [too wide](https://i.imgur.com/2ygPOPd.png) to fit on four columns on my 32:9 screen)